examples/daily-multi-translation/README.md

```diff
@@ -0,0 +1,37 @@
+# Simple Chatbot
```

Comment:
nit: update the title and the description to match the example.

Reply:
Done!